cfy_manager/components/validations.py

```diff
@@ -513,6 +513,39 @@ def validate_config_access(write_required):
                     USER_CONFIG_PATH, label))
 
 
+def _validate_not_reusing_removed_passwords():
+    # Passwords that we remove after use should not accept the 'removed' value
+    # as a password, as that may result in a remove+reinstall of the manager
+    # changing the password to a publically known value.
+    removed_value = '<removed>'
+    check_keys = {
```

Comment:
alternative would be:
```python
check_keys = [
    [POSTGRESQL_CLIENT, POSTGRESQL_PASSWORD]
]

for keys in check_keys:
    target = config
    for k in keys:
        target = target[k]
    if target == removed_value:
        problem_locations.append('.'.join(keys))
```
not sure if that's cleaner or not. Could go either way.

Or if you want to be cute:
```python
check_keys = [
    [POSTGRESQL_CLIENT, POSTGRESQL_PASSWORD]
]

for keys in check_keys:
    value = reduce(dict.get, keys, config)
    if value == removed_value:
        problem_locations.append('.'.join(keys))
```

Reply:
I like the second one, but I think the first one is probably more immediately obvious with a quick comment to avoid needing to think much about it for anyone encountering it in the middle of something else. Changed.